lib/api/interceptor.js

```diff
@@ -22,20 +22,21 @@ class XHRInterceptor {
 
       const mockedRequests = Scenarios.getCurrentMockedRequests();
       for (let mockedRequest of mockedRequests) {
-        const methodsMatch = request.method === mockedRequest.method;
-        const urlsMatch    = request.url === mockedRequest.url;
-        const paramsMatch  = request.body === mockedRequest.params;
-
-        if (methodsMatch && urlsMatch && paramsMatch && mockedRequest.active) {
-          return this.buildResponse(responder, mockedRequest.getResponse(mockedRequest.selectedStateId), request);
+        if (mockedRequest.matches({
```

Comment:
This can be written as:

```
const {url, params, method} = xhrRequest;
if (mockedRequest.matches(url, params, method) {

}
```

Which I believe will be much more readable. The current indentation is very confusing and hard to read